### PR TITLE
Created recipe for pyomeca

### PR DESCRIPTION
**Pull request by pariterre**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages
- [ ] Build number is 0
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by chrisburr** [comment]
Can you restore the example and remove the comments which were inherited

File: recipes/pyomeca/meta.yaml [hunk 1]
```diff
@@ -36,46 +35,54 @@ requirements:
...
+  doc_url: https://github.com/{{ name }}/{{ name }}
+  dev_url: https://github.com/{{ name }}/{{ name }}
 
 extra:
   recipe-maintainers:
     # GitHub IDs for maintainers of the recipe.
     # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
-    - LisaSimpson
-    - LandoCalrissian
+    - mrtnz
```
Comment (chrisburr) [inline recipes/pyomeca/meta.yaml]: @mrtnz Are you happy to be listed as a maintainer of this recipe?

File: recipes/pyomeca/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - setuptools
+    - pyyaml
+  host:
+    - python
```
Comment (chrisburr) [inline recipes/pyomeca/meta.yaml]: If you need to constrain the Python version you can use this with `noarch` packages:
```suggestion
    - python >=3
```
(edit as appropriate and include the same thing in `run`)

File: recipes/pyomeca/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  sha256: ee89099c0250e8d41984581418bef2f1e9edb9cfe6ec42753b10612ea6c29f73
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - setuptools
```
Comment (chrisburr) [inline recipes/pyomeca/meta.yaml]: These should be in `host` instead of `build`

File: recipes/pyomeca/meta.yaml (hunk 1 above)
Comment (romainmartinez) [reply recipes/pyomeca/meta.yaml]: Yes, thanks.

**Pull request merged**